src/sfctl/tests/request_generation_body_validation.py

```diff
@@ -48,3 +62,118 @@ def validate_flat_dictionary(command, actual_body, expected_body):
                                          actual_body, expected_body):
             return False
     return True
+
+
+def validate_create_application(command, actual_body, expected_body):  # pylint: disable=too-many-return-statements
```

Comment:
Could potentially serialize the bodies and compare them that way, maybe ordering the dicts first if necessary like if they could come with some difference in appearance.

Reply:
This is a great idea! It will not make a big difference in this code I believe, but in the future, I will implement it that way, thanks!